sauron/sauron.py

```diff
@@ -58,11 +55,11 @@ def init(plugin, options, configuration, **kwargs):
     plugin.log("Sauron plugin initialized")
     plugin.log(sauron_eye)
 
-
 @plugin.method("getchaininfo")
 def getchaininfo(plugin, **kwargs):
-    blockhash_url = "{}/block-height/0".format(plugin.api_endpoint)
-    blockcount_url = "{}/blocks/tip/height".format(plugin.api_endpoint)
+    blockhash_url = "{}block-height/0".format(plugin.sauron_api_endpoint)
```

Comment:
Why remove the slash? (`/`)

Reply:
I removed them because there is slash in the API address when running sauron, /api/. I thought there could be an issue when making the API calls,  ex. /block-height.  I did some testing and it does not matter, the API call works fine but I still removed it because I thought it was redundant. Should I leave them as before for readability?

Reply:
Yeah I would say generally don't change stuff unless you're sure it's an improvement.

Seems like if the user leaves off the slash in the option and there's no slash added here either, there could be an annoying error where the user doesn't realize they need to add a slash at the end of the URL they pass into the `sauron-api-endpoint` option. But if there are two slashes, I think Python figures this out. So I think there is more of a reason to leave as is than to change it.

Reply:
Yeah, that makes sense. Python does figure it out and it works just fine with two slashes. I hadn't thought  of the scenario where the user would not add the slash to the endpoint option and then none of the API calls would work. I'll add them as it was before since it would always work regardless of how the user pass the endpoint.